packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -57,6 +58,12 @@ - (instancetype)initWithURL:(NSURL *)url
 static void *playbackBufferEmptyContext = &playbackBufferEmptyContext;
 static void *playbackBufferFullContext = &playbackBufferFullContext;
 
+#if TARGET_OS_IOS
+API_AVAILABLE(ios(9.0))
+AVPictureInPictureController *_pipController;
```

Comment:
any reason this is not static?

Reply:
it is linked to your player & playerLayer. so it should not be set static.

Reply:
also the `_` prefix is only for ivars. it can get confusing when using this.